libs/sdk-ui-ext/src/internal/components/configurationControls/DataPointsControl.tsx

```diff
@@ -0,0 +1,45 @@
+// (C) 2019 GoodData Corporation
+import React from "react";
+import { WrappedComponentProps, injectIntl } from "react-intl";
+import get from "lodash/get";
+import DropdownControl from "./DropdownControl";
+
+import { dataPointsDropdownLabels } from "../../constants/dropdowns";
+import { getTranslatedDropdownItems } from "../../utils/translations";
+import { IVisualizationProperties } from "../../interfaces/Visualization";
+
+export interface IDataPointsControlProps {
+    pushData: (data: any) => any;
+    properties: IVisualizationProperties;
+    isDisabled: boolean;
+    showDisabledMessage?: boolean;
+    defaultValue?: string | boolean;
+}
+
+class DataPointsControl extends React.Component<IDataPointsControlProps & WrappedComponentProps> {
+    public static defaultProps = {
+        defaultValue: "auto",
```

Comment:
in story screenshot shows default Auto, but description mentions default Show. Please unify it in story and in code

Reply:
This is what I need to investigate, if the default behavior is right now (without the feature) really show or not.